.github/workflows/dockerise.yml

```diff
@@ -0,0 +1,68 @@
+name: Docker Build
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
```

Comment:
we need also branch `develop` under an `edge` image tag.

Reply:
so multiple dockerfiles? We should ensure we can keep branches separate right?

Reply:
no, in the same docker file.

Reply:
Example:
```yml
branches:
    - master
    - develop
```

and later


```
[ "$VERSION" == "master" ] && VERSION=latest
[ "$VERSION" == "develop" ] && VERSION=edge
```